src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -92,11 +92,11 @@ export class GcsApiService extends GcsService {
     }
   }
 
-  public cancelExportsToGCP(): Observable<boolean> {
-    const items = this.getExportPendingFiles();
+  public cancelExportsToGCP() {
+    this.cancel = true;
+    const items = this.getExportingFiles();
     if (items.length > 0) {
-      this.openModal('export-gcp-cancel', 'cancelAllExportsToGCP', Type.EXPORT_GCP);
```

Comment:
Why doesn't this open a modal dialog and send the `export-gcp-cancel` message?

Reply:
This is being handled in transferables-grid.component in order to handle cancel action in a more directly way.